src/components/Filters/BottomBar/BottomBar.tsx

```diff
@@ -1,9 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
 import { FlexContainer } from '../../FlexContainer';
+import { Spinner } from '../../Spinner';
 import { Button } from '../../Button';
 import { BottomBarProps } from './BottomBar.types';
+import {
+  getColor,
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  pxToRem,
+} from '../../../utils/helpers';
+
+const FiltersNotice = styled.div`
+  color: ${getColor('graphite2B')};
+  font-size: ${getFontSize('md')};
+  line-height: ${getLineHeight('md')};
+  margin-right: ${pxToRem(16)};
+`;
+
+const LoadingText = styled.span`
```

Comment:
`Text`, size `md`, margin `{left: 0.4}`, only font-weight should stay, since `isBold` is using `bold` not `medium`

Reply:
@ajkl2533 there is no option for white color, right?